pyproject.toml

```diff
@@ -1,10 +1,181 @@
 [build-system]
-requires = ["setuptools", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.publish.index]
```

Comment:
Perhaps move this to after [project] to keep the [tool.hatch....] settings in one group.

Reply:
Oh, yes! Thanks for noticing it.